game.tscn

```diff
@@ -1,11 +1,32 @@
-[gd_scene load_steps=3 format=3 uid="uid://ucbpmld1526y"]
+[gd_scene load_steps=5 format=3 uid="uid://d1t7eee2pxy7"]
 
+[ext_resource type="Script" path="res://game.gd" id="1_bn7oh"]
 [ext_resource type="PackedScene" uid="uid://cwibpj61yuhlv" path="res://Player/player.tscn" id="1_dve0x"]
 [ext_resource type="PackedScene" uid="uid://dqg215kxucssq" path="res://background/background.tscn" id="2_geg5b"]
 
+[sub_resource type="Curve2D" id="Curve2D_avoj2"]
+_data = {
+"points": PackedVector2Array(0, 0, 0, 0, -876, -546, 0, 0, 0, 0, 839, -549, 0, 0, 0, 0, 870, 457, 0, 0, 0, 0, -876, 453, 0, 0, 0, 0, -883, -199, 0, 0, 0, 0, -876, -552)
+}
+point_count = 6
+
 [node name="Game" type="Node2D"]
+script = ExtResource("1_bn7oh")
 
 [node name="Player" parent="." instance=ExtResource("1_dve0x")]
-position = Vector2(130, 153)
+position = Vector2(1307, 738)
+
+[node name="Path2D" type="Path2D" parent="Player"]
+curve = SubResource("Curve2D_avoj2")
+
+[node name="PathFollow2D" type="PathFollow2D" parent="Player/Path2D"]
+unique_name_in_owner = true
+position = Vector2(-876, -546)
+rotation = -0.00174927
```

Comment:
This PR works like it should, but PathFollow2D shouldn't be rotated. Reset it's rotation.